ChangeLog-Yola.md

```diff
@@ -0,0 +1,13 @@
+# Changes made by Yola
+
+Always tag using `<sem version>-patch.#`.
```

Comment:
I'd go with `-yola` rather than `-patch`. Makes it more obvious who is doing the patching.